src/codemap_details/CSS/Query.module.css

```diff
@@ -0,0 +1,70 @@
+.head {
+  margin-top: 50px;
+  letter-spacing: 0px;
+  font-size: 2.3rem;
+  font-weight: 390;
+  font-family: Arial;
```

Comment:
@prachibansal2000 Are we using Arial font for other pages?

PS: @Manvityagi If yes, let's come up with some awesome font. Arial is too mainstream.

Reply:
@aayush-05 Please check this font thing and give a common font to everyone, lets discuss this on the group.

Reply:
Moved to group